lib/src/config_document_parser.cc

```diff
@@ -33,8 +33,7 @@ namespace hocon { namespace config_document_parser {
 
     /** Parse context */
     parse_context::parse_context(config_syntax flavor, shared_origin origin, token_iterator tokens) :
-        _line_number(1), _tokens(move(tokens)), _flavor(flavor), _base_origin(move(origin)),
-        _equals_count(0) { }
```

Comment:
Erm?  I actually have no clue where those went.  I'm guessing they accidently got nuked as I went through and fixed all the LIBCPP atuff.  I'll fix.